frontend/views/pages/DesignSystem.vue

```diff
@@ -557,15 +557,15 @@ page(
           td
             .radio-wrapper
               input.radio(
-                name="v-model"
+                name='v-model'
                 type='radio'
                 checked='true'
               )
               i18n Radio label
 
             .radio-wrapper
               input.radio(
-                name="v-model"
+                name='v-model'
```

Comment:
`v-model` is a weird `name` attribute... not sure if this was intentional or not, just noticing it

Reply:
It is. renamed.